src/pytest_benchmark/table.py

```diff
@@ -22,7 +26,7 @@ def __init__(self, columns, sort, histogram, name_format, logger, scale_unit):
 
     def display(self, tr, groups, progress_reporter=report_progress):
         tr.write_line("")
-        tr.rewrite("Computing stats ...", black=True, bold=True)
```

Comment:
I don't think this should be removed - it's just one message and it give important feedback (heavy work is going on).

Reply:
I believe the issue here is that the feedback is printed for every iteration of parametrized benchmarks. I currently have my tests running with `--benchmark-disable` because these lines separate my test results from my stats by several pages' worth of scrolling. IMO, this feedback should only be output under `--benchmark-verbose`, which is what @dimrozakis 's PR is trying to achieve.

The heavy work being summarized is still work being shown. Standard output of pytest shows a few brief lines of setup, pass/fail/skipped results, & failure summaries. Moving the lengthy block of "Computing stats ... group x/y: stat (z/t)" to verbose would make the standard output more helpful as a user. Unfortunately, it is quite repetitive, especially when viewed as part of a static summary (aka pytest output piped to a file).

I'm sure the line-by-line stats computation message is very useful in a troubleshooting situation! Since I have not run into issues where it has been necessary yet, I feel like being able to turn on/off the feedback would be extremely helpful for others with my experience. The benchmark statistics themselves have been wonderful!